getting_started/publish_app.md

```diff
@@ -0,0 +1,108 @@
+# Publishing your First Project Firefly Application
+In this tutorial, we'll guide you through the following steps to give you an introduction on how to publish your first Firefly Application. 
+after you deployed your application, you need to go through below 3 steps to publish your headful app
```

Comment:
after -> After
through below 3 steps -> through the 3 steps below
your headful app -> your SPA